sefaria/utils/hebrew.py

```diff
@@ -420,12 +420,25 @@ def strip_nikkud(rawString):
 any_hebrew = regex.compile(r"\p{Hebrew}")
 any_english = regex.compile(r"[a-zA-Z]")
 
-def is_hebrew(s, heb_only=False):
-	if not heb_only and any_hebrew.search(s):
-		return True
-	elif heb_only and any_hebrew.search(s) and not any_english.search(s):
-		return True
-	return False
+
+def has_hebrew(s):
```

Comment:
can we add types to all params return -> typing? it will allow for intenseness in the IDEs to know to warn when the return param is used incorrectly